root/etc/confd/templates/config.xml.tmpl

```diff
@@ -0,0 +1,41 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I saw two bits that looked like they may have been confd parameters. Maybe it's worth owning this as a template and seeing if we can handle the override use cases this way? 90% of the changes I've done using volume mounts have related to the Views plugin configuration, which I could see handled by some kind of template-in-template embedding just as well.